common/test/acceptance/pages/lms/fields.py

```diff
@@ -133,7 +133,8 @@ def make_field_editable(self, field_id):
 
         if 'mode-placeholder' in field_classes or 'mode-display' in field_classes:
             if field_id == 'bio':
-                self.q(css='.u-field-bio > .wrapper-u-field').first.click()
+                self.wait_for_element_visibility('.u-field-bio > .wrapper-u-field', 'Bio field is visible')
+                self.browser.execute_script("$('.u-field-bio > .wrapper-u-field').click();")
```

Comment:
nit: now that you have two statements that use that same css string, it would be better to save that string in a variable and use the variable's name. This will make the code more readable and lessen the chance of a typo.